news/6390.feature

```diff
@@ -0,0 +1 @@
+Add Subversion.get_vcs_version method to return the version of the currently installed Subversion client
```

Comment:
Since this PR won't be a user-visible change, you don't need a news entry. (The news entry will come later when the feature is actually enabled.) I'll mark the PR "trivial" so CI won't require one.

Reply:
👍